sdk/core/Azure.Core/src/HttpAuthorization.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Azure.Core;
+
 namespace Azure
```

Comment:
@tg-msft is this namespace too generic?

Reply:
Customers will need to construct these so I see it as analogous to `Azure.HttpRange`.